app/test_engine/models/test_case.py

```diff
@@ -47,11 +47,14 @@ def default_test_parameters(cls) -> dict[str, Any]:
 
     @property
     def test_parameters(self) -> dict[str, Any]:
-        if self.config.test_parameters is None:
+        config_dict = cast(dict, self.config)
+
+        if config_dict and config_dict.get("test_parameters") is None:
             return self.default_test_parameters()
         else:
+            test_parameters = config_dict.get("test_parameters")
```

Comment:
this conditional is wrong. if `config_dict` is `None` it will execute `config_dict.get()`...

Reply:
This fix will be in a different PR

Reply:
OK!